src/main/java/cz/brmlab/yodaqa/pipeline/solrdoc/SolrDocPrimarySearch.java

```diff
@@ -155,12 +163,19 @@ protected void documentToAnswer(JCas jcas, SolrDocument doc,
 		jcas.setDocumentText(title.replaceAll("\\s+\\([^)]*\\)\\s*$", ""));
 		jcas.setDocumentLanguage("en"); // XXX
 
+		AnswerSourceEnwiki ac = new AnswerSourceEnwiki("document title", (title != null ? title : ""), id);
```

Comment:
For the other enwiki sources, we have string constants in the class.  We should do it here too.

The reasoning is that these are machine-readable identifiers, not meant necessarily directly for the end-user, they are part of our API.  Therefore, all possible values should be at a single place to make the code self-documenting.